### PR TITLE
Add Jitbit Helpdesk

### PR DESCRIPTION
**Pull request by maxt3r**

### CODE REVIEW

**Comment by maxt3r** [comment]
How to check why it failed?

File: _data/other.yml [hunk 1]
```diff
@@ -304,6 +304,14 @@ websites:
       img: instapaper.png
       tfa: No
 
+    - name: Jitbit Helpdesk
+      url: https://www.jitbit.com/helpdesk/
+      img: jitbit.png
+      tfa: Yes
+      sms: No
```
Comment (mxxcon) [inline _data/other.yml]: No need for explicit `no` tag.

**Review by mxxcon — CHANGES_REQUESTED** [review]
At this point in time we do not list self-hosted software since that brings too many variables in how to do display such data.

**Comment by maxt3r** [comment]
I changed the link to the hosted version (we have both)

**Review by mxxcon — APPROVED** [review]

**Review by brycx — APPROVED** [review]

**Pull request merged**